repos/fdbt-site/src/pages/api/selectSalesOfferPackage.ts

```diff
@@ -83,7 +87,28 @@ export const sanitiseReqBody = (
         errors,
     };
 };
-
+export const getProductsByValues = (
+    ticket: TicketWithIds | undefined,
+    multipleProductAttribute: MultipleProductAttribute | MultipleProductAttributeWithErrors | undefined,
+    pricePerDistanceName: string,
```

Comment:
If the type we have here is string, and we're checking if its defined lower down, that insinuates we're passing an empty string to it sometimes. Could we change this type to string | undefined?